src/NUnitFramework/framework/Internal/Filters/AndFilter.cs

```diff
@@ -32,9 +31,27 @@ internal sealed class AndFilter : CompositeFilter
         public override bool Pass( ITest test, bool negated )
         {
             if (negated)
-                return Filters.Any(f => f.Pass(test, negated));
```

Comment:
these LINQ `Any` can be a beast as it used to create `Func<>`  instance, new versions are more verbose, but a lot faster